docs/setup-cluster/k8s/setup-multiple-resource-managers.rst

```diff
@@ -0,0 +1,167 @@
+The Cluster Admin role is required for configuring multiple resource managers.
+
+.. attention::
+
+   **Naming and Rules**
+
+   -  Each resource manager under ``additional_resource_managers`` must have a unique name; failure
+      to do so will cause the cluster to crash.
+
+   -  Ensure each additional resource manager has at least one resource pool defined.
+
+   -  Resource pool names must be unique across the cluster to prevent crashes.
+
+.. note::
+
+   The default resource manager is available by default and does not require a specific name.
+
+#. Locate the ``resource_manager`` section in the :ref:`master configuration <master-config-reference>`
+   yaml file. This represents the default resource manager.
+#. Add ``additional_resource_managers`` under the ``resource_manager`` to configure extra resource
+   managers.
+#. Under ``additional_resource_managers``, define ``resource_pools`` for each additional resource
+   manager.
+
+
+Example: Master Configuration (devcluster)
+==========================================
+
+Follow this example to create as many resource managers (clusters) as needed.
+
+-  For each cluster, note each ``kubeconfig`` location for the cluster (this is where the credentials
+   are found).
+
+-  Copy or modify the default devcluster template at ``tools/devcluster.yaml``.
+
+-  In the copied ``devcluster.yaml`` file, under the ``master configuration``, set one of your
+   resource managers as the default, and the rest under ``additional_resource_managers``:
+
+.. code:: yaml
+
+   resource_manager:
+   type: kubernetes
+   name: default-rm # optional, should match the name of your default RM/cluster
+   ... add any other specs you might need ...
+   additional_resource_managers:
+   - resource_manager:
+      type: kubernetes
+      name: additional-rm # should match the name of your other RM(s)
+      kubeconfig_path: <whatever-path-your-rm-config-is-like ~/.kubeconfig>
+      ... add whatever other specs you might need ...
+      resource_pools:
+         - pool_name: <your-rm-pool-name>
+
+-  Run the new devcluster: ``devcluster -c <path-to-modified-devcluster>``.
+
+Example: Master Configuration (Helm)
+====================================
+
+To deploy Multi-RM on Kubernetes through a Helm chart, the cluster administrator must load
+the credentials for each additional cluster through a Kubernetes secret. Follow these steps for each
+additional resource manager you want to add, and then apply the Helm chart once at the end. Let
+``rm-name`` be the same as the “cluster name” for a given cluster.
+
+- Set up your additional clusters. These can be from the same or different clouds (e.g., GKE, AKS, EKS).
+- Gather the credentials for each cluster.
+
+For example:
+
+.. code:: bash
+
+   # for AKS az aks get-credentials --resource-group <resource-gp-name> --name <rm-name> # for
+   GKE gcloud container clusters get-credentials <rm-name>
```

Comment:
for AKS 
az aks get-credentials --resource-group <resource-gp-name> --name <rm-name> 

for GKE 
gcloud container clusters get-credentials <rm-name>